src/machines/java/com/enderio/machines/common/blockentity/XPVacuumBlockEntity.java

```diff
@@ -66,7 +66,7 @@ public void saveAdditional(CompoundTag pTag) {
     @Override
     public void load(CompoundTag pTag) {
         super.load(pTag);
-        fluidTank.readFromNBT(pTag.getCompound("Fluids"));
+        fluidTank.readFromNBT(pTag.getCompound("tank"));
```

Comment:
Why was this modified?

Reply:
Isn't the NBT id supposed to be the same in saveAdditional() and load() ? It's "tank" in saveAdditional().

Reply:
Dang yeah it is, nice catch!